internal/commands/test.go

```diff
@@ -177,11 +177,19 @@ func NewTestCommand(ctx context.Context) *cobra.Command {
 				return fmt.Errorf("build store: %w", err)
 			}
 
-			namespace := viper.GetString("namespace")
+			var namespaces []string
+			if viper.GetBool("all-namespaces") {
+				namespaces, err = policy.GetNamespaces(regoFiles, compiler)
+				if err != nil {
+					return fmt.Errorf("build compiler: %w", err)
```

Comment:
`build compiler` should probably be `get namespaces` here

Reply:
Good catch! Should be fixed now. Thanks for taking the time reviewing.